torchvision/models/quantization/mobilenetv3.py

```diff
@@ -135,20 +136,30 @@ def _mobilenet_v3_model(
         if "backend" in weights.meta:
             _ovewrite_named_param(kwargs, "backend", weights.meta["backend"])
     backend = kwargs.pop("backend", "qnnpack")
+    fx_mode = kwargs.pop("fx_mode", False)
 
     model = QuantizableMobileNetV3(inverted_residual_setting, last_channel, block=QuantizableInvertedResidual, **kwargs)
     _replace_relu(model)
 
     if quantize:
-        model.fuse_model(is_qat=True)
-        model.qconfig = torch.ao.quantization.get_default_qat_qconfig(backend)
-        torch.ao.quantization.prepare_qat(model, inplace=True)
+        qconfig = torch.ao.quantization.get_default_qat_qconfig(backend)
+        if fx_mode:
+            qconfig_dict = {"": qconfig}
+            model = torch.ao.quantization.quantize_fx.fuse_fx(model)
+            model = torch.ao.quantization.quantize_fx.prepare_qat_fx(model, qconfig_dict)
+        else:
+            model.qconfig = qconfig
+            model.fuse_model(is_qat=True)
```

Comment:
You've switched the order of `qconfig` assignment and `fuse_model()` call. Is this expected to have an effect?

Reply:
I checked with @jerryzh168. It should not change any behavior

Reply:
Right. I just switched it to make the code look more consistent with the FX branch. I don't think it should have any change in behavior.

Reply:
(This change was reverted since we don't need to explicitly fuse in FX mode)